File: ObjectiveC_Code/ObjectiveC_Code/Views/VideoCollectionViewCell.m
//
//  VideoCollectionViewCell.m
//  ObjectiveC_Code
//
//  Created by Mallikarjun on 08/06/20.
//  Copyright © 2020 Mallikarjun. All rights reserved.
//

#import "VideoCollectionViewCell.h"
#import <QuartzCore/QuartzCore.h>

@implementation VideoCollectionViewCell

- (void) image:(NSString*)image title:(NSString*)title  time:(NSString*)time{
    
  
    self.videoImg.layer.cornerRadius = 5;
    self.videoImg.layer.masksToBounds = YES;
    
    self.videoImg.image = [UIImage imageNamed:image];
    self.videoTitleLabel.text = title;
    self.videoTimeLabel.text = time;
}


@end

File: SwiftCode/SwiftCode/Views/VideoCollectionViewCell.swift
//
//  VideoCollectionViewCell.swift
//  SwiftCode
//
//  Created by Mallikarjun on 08/06/20.
//  Copyright © 2020 Mallikarjun. All rights reserved.
//

import UIKit

class VideoCollectionViewCell: UICollectionViewCell {
    
    @IBOutlet weak var videoImg: UIImageView!
    @IBOutlet weak var titleLabel: UILabel!
    @IBOutlet weak var timeLabel: UILabel!
    
    func fillDetails(_ image:String, _ title:String, _ time:String){
        
        videoImg.layer.cornerRadius = 5
        
        videoImg.image = UIImage(named: image)
        titleLabel.text = title
        timeLabel.text = time
    }
    
}
